lua/telescope/builtin/__git.lua

```diff
@@ -332,7 +332,7 @@ git.status = function(opts)
     end
 
     return finders.new_table {
-      results = output,
+      results = vim.split(output[1], " ", { trimempty = true }),
```

Comment:
For some reason, GitHub cuts off the preview after the NUL character. It's shown correctly here: https://github.com/nvim-telescope/telescope.nvim/blob/889ef096d11fd4b9476c0409be544a9d8460a2b8/lua/telescope/builtin/__git.lua#L335